documentation/quarkus/getting-started-quarkus.asciidoc

```diff
@@ -0,0 +1,235 @@
+. Difference to SpringData + 
+If you have already worked with SpringData this is also partially supported with Quarkus, this is explained in more detail in this link:../guide-repository.asciidoc[SpringData Guide].
+
+
+.Database Integration 
+. Quarkus zero config dev mode + 
+Starting with database implementation in Quarkus, we recommend for beginners to use the https://quarkus.io/guides/datasource#dev-services[DEV mode Zero Config Setup (Dev Services)] this is especially great for testing the code without a database set up. 
+Quarkus does all the work for you and configures a database and creates the database and tables(schemas) for you.
+.. Configuration Properties + 
+A list of all database configuration properties for the https://quarkus.io/guides/datasource#configuring-dev-services[Dev services]
+. Integrate a simple Hibernate ORM database + 
+The zero config setup only works with the Dev mode, it's comfortable in the first phases of the creation of your service but if the goal is to also get a deployable version, you have to create your own database and integrate it.
+This Quarkus guide shows, how to integrate a https://quarkus.io/guides/hibernate-orm[Hibernate ORM database] with an example service.
+.. Configuration list for JDBC + 
+A list of all configuration that is possible with a https://quarkus.io/guides/datasource#jdbc-configuration[JDBC configuration properties]
+. Reactive CRUD application with Panache + 
+Quarkus unifies reactive and imperative programming.
+Reactive is an architectural principle to build robust, efficient, and concurrent applications. 
+An introduction into reactive and how Quarkus is enabling it follow this https://quarkus.io/guides/quarkus-reactive-architecture[Quarkus reactive architecture] article and also the https://quarkus.io/guides/getting-started-reactive[reactive quickstart].
+To get started with reactive and implement reactive methods you can follow the https://quarkus.io/guides/hibernate-reactive-panache[Quarkus reactive guide].
+The reactive guide is using the Quarkus based implementation of a Hibernate ORM called Panache. 
+That implementation is not our first choice with devon4j and therefore not part of our recommendations, but to understand the reactive guide you can read the https://quarkus.io/guides/hibernate-orm-panache[Hibernate ORM with Panache guide] first to prevent possible problems following the guide.
+
+[NOTE]
+====
+You need an installed Docker version for the https://quarkus.io/guides/datasource#dev-services[zero config setup].
+==== 
+
+
+.Database Migration
+. Migration guide
+For schema-based databases, we recommend migrating databases with Flyway.
+In that case there is our link:../guide-database-migration.asciidoc[Quarkus non-specific migration guide] can give you an overview if you are not familiar with migration.
+.. Flyway guide for Quarkus
```

Comment:
?

Reply:
I do not see the problem here, can you explain it to me?

Reply:
what is the `..`?

Reply:
This will generate a List with a List like that 
1. Migration guide
   a. Flyway guide
I wanted to arrange the guides this way because you should have read the first guide before working on the second guide.

Reply:
ok ok, you are totally right :) thanks I was not getting it